pig/src/main/resources/interpreter-setting.json

```diff
@@ -15,6 +15,12 @@
         "propertyName": "zeppelin.pig.includeJobStats",
         "defaultValue": "false",
         "description": "flag to include job stats in output"
+      },
+      "zeppelin.pig.execType": {
+        "envName": "SPARK_MASTER",
+        "propertyName": "SPARK_MASTER",
```

Comment:
Or, do you mean for this to be
```
+      "SPARK_MASTER": {
 +        "envName": "SPARK_MASTER",
 +        "propertyName": "SPARK_MASTER",
```
since `zeppelin.pig.execType` is already on the list?